### ISSUE TITLE
IOLinc Not Showing Properly

### ISSUE THREAD

**Issue by shbatm** [shbatm/hacs-isy994#2]
1. haven’t changed the sensor string, in fact these aren’t even referenced anywhere in the hassio config. I only use them in floorplan and homekit.
2. Unlike my thermostat device, these aren’t grouped since at one point I had them in separate folders in the ISY UI.

My other I/O Linc that is acting the same way doesn’t have sensor in the name and is only mentioned in customize.yaml (which of course isn’t working since there I refer to it as a switch and it’s coming in as a sensor):

```
switch.projector_screen_trigger:
  hidden: true
```

The developer tools show:

|Entity|State|Attributes|
| --- | --- | --- |
|sensor.projector_screen|0.0|unit_of_measurement: 100 friendly_name: Projector Screen|
|sensor.projector_screen_trigger|0.0|unit_of_measurement: 100 friendly_name: Projector Screen Trigger|

And the nodes look like:

Copy to clipboard

```
  <node flag="128" nodeDefId="BinaryControl_ADV">
    <address>28 C0 72 1</address>
    <name>Projector Screen Trigger</name>
    <parent type="3">62374</parent>
    <type>7.0.65.0</type>
    <enabled>true</enabled>
    <deviceClass>0</deviceClass>
    <wattage>0</wattage>
    <dcPeriod>0</dcPeriod>
    <startDelay>0</startDelay>
    <endDelay>0</endDelay>
    <pnode>28 C0 72 1</pnode>
    <property id="ST" value="0" formatted="Off" uom="100"/>
  </node>
  <node flag="0" nodeDefId="RelayLampOnly">
    <address>28 C0 72 2</address>
    <name>Projector Screen</name>
    <parent type="3">62374</parent>
    <type>7.0.65.0</type>
    <enabled>true</enabled>
    <deviceClass>0</deviceClass>
    <wattage>0</wattage>
    <dcPeriod>0</dcPeriod>
    <startDelay>0</startDelay>
```

#### Reference Forum Topic ####
https://community.home-assistant.io/t/insteon-thermostat-2441th-for-isy/97821/62

**Comment by shbatm** [comment]
This should be fixed now with v1.0.3.

**Comment by shbatm** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[shbatm/PyISY] Add uom to send_cmd (#8)](https://trello.com/c/P3sq5SZw/237-shbatm-pyisy-add-uom-to-sendcmd-8)
